netlify-cleanup.sh

```
@@ -0,0 +1,47 @@
+echo 
+if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+  echo "Operation cancelled."
+  exit 1
+fi
+
+# For each merged branch, try to unpublish its deployment
+for branch in $MERGED_BRANCHES; do
+  echo "Processing branch: $branch"
+  
+  # Get the deploy ID for this branch
+  DEPLOY_ID=$(netlify api listSiteDeploys --data '{"site_id": "'$(netlify api getSite | jq -r '.id')'"}' | jq -r '.[] | select(.branch=="'$branch'") | .id' | head -n1)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)